geography/README.md

```diff
@@ -0,0 +1,150 @@
+# Mobility Data Specification: Geography
+
+This specification contains a collection of RESTful APIs used to read Geographies (descriptions of geographical information, e.g. multi-polygons, currently represented via GeoJSON).
```

Comment:
Are geographies always single or multi polygons?  They will never be a line or point right?  Either way we should clarify this somewhere in the spec.  For an example, #480 assumes only polygons.

Reply:
`FeatureCollection` is the highest-level concept.  You can include lines and points but they are no-ops because they can't contain points.  We will revisit once we have a curb spec.